package-lock.json

```
@@ -1,12 +1,12 @@
 {
   "name": "orion",
-  "version": "3.0.2",
+  "version": "3.0.4",
   "lockfileVersion": 2,
   "requires": true,
   "packages": {
     "": {
       "name": "orion",
-      "version": "3.0.2",
+      "version": "3.0.4",
```

```suggestion
  "version": "3.0.5",
```
As 3.0.4 is already out